app/models/ambiguous_postcode_exception.rb

```diff
@@ -0,0 +1,9 @@
+class AmbiguousPostcodeException < StandardError
+  attr_reader :postcode, :addresses
+
+  def initialize(postcode, addresses)
+    super()
+    @postcode = postcode
```

Comment:
Do we need to include `postcode` in the error class, as we only seem to [use addresses](https://github.com/alphagov/imminence/pull/862/commits/80c36e80ae22803b38eb566738a6ae168c8b80b5#diff-3b3db6d3128e78205dd32fc41491b1aed2543ce04ef045b21560bd5176a062cbR65)?